src/libraries/LiquidityMath.sol

```diff
@@ -17,4 +19,19 @@ library LiquidityMath {
             }
         }
     }
+
+    /// @notice Flips the sign of a liquidity delta if a condition is true
+    /// @dev Equivalent to `res = flip ? -liquidityDelta : liquidityDelta;`
+    /// @dev `liquidityDelta` should never be `type(int128).min`
+    /// @param liquidityDelta The liquidity delta to potentially flip
+    /// @param flip Whether to flip the sign of the liquidity delta
+    /// @return res The potentially flipped liquidity delta
+    function flipLiquidityDelta(int128 liquidityDelta, bool flip) internal pure returns (int128 res) {
+        assembly {
```

Comment:
memory safe assembly comment?

Reply:
That wouldn't be necessary since the assembly block contains no memory opcodes.